### DISCUSSION TITLE
How can we Integrate a Azure deveops UI into Serenity ?

### DISCUSSION THREAD

**Discussion by rakeshhkumarofficial** [serenity-is/Serenity#7280] [Help]
How can we Integrate a Azure deveops UI into Serenity and build a simple demo app using it? Please let me steps to integrate

**Comment by edwardch** [comment]
Are you trying to embed something from DevOps into your app?
Or are you trying to build your app in a DevOps pipeline?
Or something else?

**Reply by rakeshhkumarofficial** [reply]
I want to integrate this azure-deveops-ui (https://developer.microsoft.com/en-us/azure-devops/components) components in my serenity project and use it. Please let me know the steps to integrate and how can i use azure deveops ui components in my app.
